src/tests/equality_with_units.py

```diff
@@ -1,9 +1,11 @@
 """Functions for using pytest with units."""
```

Comment:
I think this should go into src/amuse/support/testing, so that it'll get included into the framework package.